### PR TITLE
README for #5

### PR DESCRIPTION
**Pull request by barbeque**
I created a brief README though there really isn't much to go on (maybe the options for the *gen scripts?)

I started work on some PC98 _floppy_ image tools over at https://github.com/barbeque/pc98-disk-tools, would you be possibly interested in collaborating on more stuff like this in the future?

### CODE REVIEW

**Comment by tsdko** [comment]
Thanks for the contribution! Turns out I've already started writing a README in my local repo but haven't pushed it, I'll try to merge the two versions since mine is much more brief. I've actually tried to implement d88 as one of the formats in this project (it wasn't really supposed to be limited to HDD images), but something didn't exactly work right and I didn't bother looking into what exactly was wrong at the time. I mean, there are image conversion tools out there already, so I didn't think it would matter anyway.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+See [`formats`](formats/) for the various format structures.
+
+## Tools
+### `hdddump.py`
+Display header information from an HDD hard drive image file.
+
+Verifies magic values in header.
+
+### `hddgen.py`
+Creates a new blank HDD image file to the user's specifications.
```
Comment (tsdko) [inline README.md]: All `*gen.py` scripts actually only create image *headers*, not the images themselves. I'd imagine most emulators would expect some blank space after the header.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### `nhddump.py`
+Display header information from an NHD hard drive image file.
+
+Verifies magic values in header.
+
+### `nhdgen.py`
+Creates a new blank NHD image file to the user's specifications.
+
+### `pbr_dump.py`
+Tries to identify partitions in a disk image.
```
Comment (tsdko) [inline README.md]: `pbr_dump` dumps the boot sector of the first FAT partition found in the file.

**Comment by barbeque** [comment]
I made some tweaks for your feedback, hopefully that doesn't make the job of merging your side more difficult.

**Pull request merged**